pkg/entrypoint/metricsbridge/metricsbridge.go

```diff
@@ -286,14 +294,31 @@ func start(cfg *cmdConfig) error {
 	logrus.SetFormatter(&logrus.TextFormatter{FullTimestamp: true})
 	log := logrus.NewEntry(logrus.StandardLogger())
 
+	log.Info("starting metrics endpoint")
+	l, err := net.Listen("tcp", fmt.Sprintf(":%d", cfg.httpPort))
+	if err != nil {
+		return err
+	}
+
+	mux := &http.ServeMux{}
+	mux.Handle("/healthz/ready", http.HandlerFunc(readyHandler))
```

Comment:
Why `/healthz/ready`? Isn't `/healthz` more standard?

Reply:
Yes `/healthz` is more standard but `/healthz/ready` is the convention we've used so far for the ready handlers for the `canary`, `sync` and `azure-controllers` pods.

Reply:
Eh, best to remain consistent then.